tests/sentry/ingest/test_transaction_clusterer.py

```diff
@@ -72,26 +73,26 @@ def test_collection():
     for name in set_entries2:
         assert name.startswith("tx-project2-")
 
-    project3 = Project(id=103, name="project3", organization_id=1)
+    project3 = Project(id=103, name="project3", organization=Organization(pk=66))
```

Comment:
The extra feature check now uses `project.organization` within `_store_transaction_name`, so we need to provide it in the tests.